.github/workflows/deploy-preview.yml

```diff
@@ -17,6 +17,16 @@ jobs:
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Install npm packages
+        run: |
+          npm ci
+
+      # TODO: any other node-y things to do prior to build/deploy? 'npm run styles', etc.?
```

Comment:
We could run `npm run styles` but have so far been pushing up the CSS files with PRs. We could add it if we want to stop tracking the generated CSS files as well. @flynnduism any preferences here?

Reply:
Created https://github.com/fermyon/developer/issues/609 as a potential follow-up.